cloudformation/secure-contact.template.yaml

```diff
@@ -0,0 +1,131 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Run the SecureContact service in AWS
+
+Parameters:
+  VpcId:
+    Description: The VPC in which SecureContact will run
+    Type: AWS::EC2::VPC::Id
+
+  AMI:
+    Description: Base AMI for SecureContact instances
+    Type: AWS::EC2::Image::Id
+
+  SSHAccessCidr:
+    Type: String
+    Description: A CIDR from which SSH access to the instance is allowed
+    AllowedPattern: ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}/[0-9]{1,2}$
+    ConstraintDescription: Parameter should be a CIDR block e.g. "1.2.3.4/32"
+
+  KeyName:
+    Description: An ssh keypair to put on the instance
+    Type: AWS::EC2::KeyPair::KeyName
+
+  AccountId:
+    Type: String
+    Description: The account that this application will use
+
+  DataBucketName:
+    Description: Name to use for the S3 bucket that data is uploaded to
+    Type: String
+
+Mappings:
+  Constants:
+    App:
+      Value: secure-contact
+    Stack:
+      Value: infosec
```

Comment:
Should probably add a `Stage` tag to everything to make it play nice with `prism`